classes/snake.js

```diff
@@ -14,6 +14,10 @@ class Snake{
             return coord.x === c.x && coord.y === c.y;
         });
     }
+    // Get the length of the snake
+    length() {
```

Comment:
This doesn't need to be a function, could just be a property set in the constructor
`this.length = this.body.length;`